util/mklink.pl

```diff
@@ -55,8 +55,10 @@
 foreach $file (@files) {
     my $err = "";
     if ($symlink_exists) {
-	unlink "$from/$file";
-	symlink("$to/$file", "$from/$file") or $err = " [$!]";
+        if (!-l "$from/$file") {
```

Comment:
I rather wonder what this new condition is for.  What was the case that had you make this change?